src/components/tx/confirmation-views/utils.tsx

```diff
@@ -0,0 +1,11 @@
+import { TransactionInfoType } from '@safe-global/safe-gateway-typescript-sdk'
+import type { NarrowConfirmationViewProps } from './types'
+import type { SettingsChange } from '@safe-global/safe-gateway-typescript-sdk'
+import SettingsChangeComponent from './SettingsChange'
+
+export const getConfirmationViewComponent = (txType: TransactionInfoType, props: NarrowConfirmationViewProps) => {
+  if (txType === TransactionInfoType.SETTINGS_CHANGE)
+    return <SettingsChangeComponent txDetails={props.txDetails} txInfo={props.txInfo as SettingsChange} />
+
+  return null
+}
```

Comment:
That's not really a util. I would keep this mapper directly in `cofirmations-views/index.tsx`.